### PR TITLE
Feature/condo/doma 1626/meter schema improvments

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
The previous way of validating user with type resident rights did not work (worked in tests, but not on a real device).
Rewrote access rights normally.

### CODE REVIEW

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/meter/schema/MeterReading.js [hunk 1]
```diff
@@ -41,65 +41,67 @@ const MeterReading = new GQLListSchema('MeterReading', {
...
-                    if (operation === 'create') {
-                        const value = Number(resolvedData[fieldPath])
-                        const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
-
-                        if (lastMeterReading && lastMeterReading.value3 > value)
-                            addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
-                    }
-                },
-            },
+            // hooks: {
```
Comment (sitozzz) [inline apps/condo/domains/meter/schema/MeterReading.js:78]: You can delete these lines if they are no longer needed?

Comment (nomerdvadcatpyat) [reply apps/condo/domains/meter/schema/MeterReading.js:78]: We will most likely come back to this. I thought it would be better to comment it out

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/meter/schema/MeterReading.js [hunk 2]
```diff
@@ -41,65 +41,67 @@ const MeterReading = new GQLListSchema('MeterReading', {
...
-                        const lastMeterReading = await getLastBillingAccountMeterReading(context, resolvedData)
-
-                        if (lastMeterReading && lastMeterReading.value1 > value)
-                            addFieldValidationError(`${VALUE_LESS_THAN_PREVIOUS_ERROR}value] Meter reading value less than previous`)
-                    }
-                },
-            },
+            // For now we have decided to refuse the validation of new readings relative to previous ones
+
+            // hooks: {
```
Comment (toplenboren) [inline apps/condo/domains/meter/schema/MeterReading.js:46]: I'd rather remove the commencted code from repo, you can still get it from git anyways

File: apps/condo/domains/meter/schema/MeterReading.js (hunk 1 above)
Comment (toplenboren) [reply apps/condo/domains/meter/schema/MeterReading.js:78]: Please add TODO with task on either removing or restoring the commentary blocks :)

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/meter/schema/MeterReading.js (hunk 1 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/meter/schema/MeterReading.js:78]: Removed all comented lines from this pr in #823

**Pull request merged**